### DISCUSSION TITLE
WMTJDT1

### DISCUSSION THREAD

**Discussion by LivingBuzzard55** [iptv-org/community#935] [Show and tell]
### Channel ID

WMTJDT1.pr

### Stream URL (optional)

https://wmtjdt.lls.pbs.org/out/v1/c5c8fd0b62f944268731ba15ffbaa39c/hls-drm.m3u8
https://wmtjdt.lls.pbs.org/out/v1/b0118cb3654b4153a07e6601db780224/dash-drm.mpd

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by dtngfan85** [comment]
The PBS streams are DRM protected

**Comment by freearhey** [comment]
https://wmtjdt.lls.pbs.org/out/v1/c5c8fd0b62f944268731ba15ffbaa39c/hls-drm.m3u8 (offline)

![image](https://github.com/user-attachments/assets/39c2c4a9-5693-482c-9fa0-203330501ebf)
